apps/am/am-judicial-booking-service/ithc.yaml

```diff
@@ -5,5 +5,24 @@ metadata:
 spec:
   values:
     java:
+      keyVaults:
+        am:
+          secrets:
+            - name: am-judicial-booking-service-s2s-secret
+              alias: AM_JUDICIAL_BOOKING_SERVICE_SECRET
+            - name: judicial-booking-service-POSTGRES-PASS-V15
+              alias: JUDICIAL_BOOKING_SERVICE_POSTGRES_PASS
+            - name: judicial-booking-service-POSTGRES-USER-V15
+              alias: JUDICIAL_BOOKING_SERVICE_POSTGRES_USER
+            - name: judicial-booking-service-POSTGRES-HOST-V15
+              alias: JUDICIAL_BOOKING_SERVICE_POSTGRES_HOST
+            - name: app-insights-connection-string
+              alias: app-insights-connection-string
+            - name: judicial-booking-service-IDAM-CLIENT-SECRET
+              alias: JUDICIAL_BOOKING_IDAM_CLIENT_SECRET
+            - name: judicial-booking-service-IDAM-ADMIN-SECRET
+              alias: JUDICIAL_BOOKING_IDAM_ADMIN_PASSWORD
```

Comment:
can you rename to match other naming convention please

Reply:
these variable names are the ones currently use in the app i believe